src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -292,6 +292,10 @@ class LedgerMaster : public AbstractFetchPackContainer
     std::optional<LedgerIndex>
     minSqlSeq();
 
+    // Iff a txn exists at the specified ledger and offset then return its txnid
+    std::optional<uint256>
+    txnIDfromIndex(uint32_t ledgerSeq, uint32_t txnIndex);
```

Comment:
This should be named `txnIdFromIndex`. The "d" in "Id" should not be capitalized, and the "F" in "From" should be.

Reply:
@thejohnfreeman Can you review this again? I made a mistake. Richards code was correct and my tests should have just reflected that.

If the `args.hash` is UB then we need to exit early and then searchAll needs to be/will always be "unknown".

Reply:
A check was added immediately above:
https://github.com/XRPLF/rippled/blob/80f308f62ecb2e55208af33bbddb2260c9ece3de/src/ripple/rpc/handlers/Tx.cpp#L220-L225

Reply:
I fixed the function name of the test so that rather than containing the substring `IDfrom` it now contains the substring `IdFrom` @thejohnfreeman